ui/component/claimPreviewSubtitle/view.jsx

```diff
@@ -34,9 +42,18 @@ function ClaimPreviewSubtitle(props: Props) {
           <UriIndicator uri={uri} link />{' '}
           {!pending && claim && (
             <>
-              {isChannel &&
-                type !== 'inline' &&
-                `${claimsInChannel} ${claimsInChannel === 1 ? __('upload') : __('uploads')}`}
+              {isChannel && type !== 'inline' && (
+                <>
+                  <br />
```

Comment:
I believe we don't want `<br />`s in the code, and there was a task to remove all of them in the past.